pkg/server/httpconfig.go

```diff
@@ -70,3 +72,14 @@ func (hc HTTPConfig) HTTPTarget() string {
 }
 
 func (hc HTTPConfig) HTTPMetricsTarget() string { return hc.host + ":" + strconv.Itoa(hc.metricsPort) }
+
+func (hc HTTPConfig) HasTLS() bool {
```

Comment:
shoot I missed these tests, can you update `httpconfig_test.go` and `grpcconfig_test.go`?